packages/@aws-cdk/aws-stepfunctions-tasks/package.json

```diff
@@ -179,7 +179,31 @@
       "props-default-doc:@aws-cdk/aws-stepfunctions-tasks.TransformOutput.encryptionKey",
       "props-default-doc:@aws-cdk/aws-stepfunctions-tasks.TransformResources.volumeKmsKeyId",
       "docs-public-apis:@aws-cdk/aws-stepfunctions-tasks.VpcConfig",
-      "props-default-doc:@aws-cdk/aws-stepfunctions-tasks.VpcConfig.subnets"
+      "props-default-doc:@aws-cdk/aws-stepfunctions-tasks.VpcConfig.subnets",
```

Comment:
Oh one more thing, sorry: you shouldn't really be ignoring all these warnings, you should be fixing them instead.

Why not add documentation on default behavior to all props?

Reply:
I added documentation on the default values where the EMR __RunJobFlow__ API documentation gave information on the behavior of the defaults, or where I made a sane choice on behalf of the user. But the docs don't provide info on the default behavior of everything. SubnetId and SubnetIds are good examples. Neither is required and the docs don't mention what happens when none is provided, but a Subnet is somehow picked.

If you prefer, I can fix the errors by indicating that `None` is the default for the interface, but won't be able to document the underlying behavior.

Reply:
We can probably guess at the behavior, but if you don't want to do that, let's make our lack of knowledge explicit using something like:

```
@default - EMR will automatically select a subnet (details are service-specific)
```

Maybe you can think of a better phrasing.

Reply:
linter errors fixed